src/System.Net.Http/tests/StressTests/HttpStress/global.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
A comment here would be helpful

Reply:
json doesn't support comments, unfortunately